isc24_tutorial/isc24_tutorial_proposal.md

```diff
@@ -0,0 +1,164 @@
+in-person or online workshops.
+
+After attending this tutorial, participants will:
+
+- Have increased confidence in teaching interactive workshops in online and in-person settings.
+- Be familiar with important pedagogical concepts and scientific findings about how people learn new skills.
+- Appreciate the importance of frequent hands-on practice in building skill.
+- Understand how learning can be improved by following certain best practices in interactive teaching.
+- Know how to backwards-design training material based on intended learning outcomes.
+- Know about important differences between online and in-person training and how inherent challenges in online training can be overcome.
+- Be familiar with a technical approach to collaboratively developing open source lessons based on a template. 
+
+
+**Targeted audience**
+
+Teaching is a profession, but also something that everyone needs to be able to
+do to some degree, since everyone has their own personal specialties they will
+either teach or mentor.
+
+This course can be relevant for different learner personas:
+
+- **You run a practical teaching program** at your institution (for example as part of a research computing group) and would like to learn best practices for collaborative teaching of technical topics, so that you aren’t re-inventing the same thing over and over again.
+- **You are a technical specialist** who is frustrated with the way you currently try to teach others who need to use your software or infrastructure. You can’t spend too much time to become a professional, but you know you need something more than what you’ve been doing. Thus, you would like to adopt some of the best practices of designing and teaching interactive, hands-on workshops.
+- **You’ve been teaching alone**, but would like to join a collaboration network for more co-teaching and to reduce the amount of duplication of effort.
+- **You are interested in teaching CodeRefinery/ENCCS/... lessons**, and would like a comprehensive kick-start to how these workshops work, either to join and teach with us, or teach our lessons independently.
+
+**Detailed outline of the tutorial (with time slots)**
+
+| Time | Topic |
+| ---- | ----- |
+|09:00-09:30 | Welcome and introduction|
+|09:30-09:45 | Training providers - EuroCC, CodeRefinery, Carpentries, ...|
+|09:45-10:00 | Available public training resources|
+|10:00-10:10 | Break|
+|10:10-11:00 | Acquisition of skill|
+|11:00-11:30 | Break|
+|11:30-12:10 | Lesson design|
+|12:10-13:00 | Good interactive teaching practices|
+|13:00-14:00 | Lunch break|
+|14:00-14:40 | Teaching mechanics|
+|14:40-15:20 | Collaborative lesson development|
+|15:20-15:30 | Break|
+|15:30-16:00 | Practice: teaching and giving feedback|
+|16:00-16:30 | Break|
+|16:30-17:30 | Practice: teaching and giving feedback|
+|17:30-18:00 | Recap, final discussion and where to go from here|
+
+**Description of hands-on activities, if applicable**
```

Comment:
i think we'll have to figure that out later and just go for this schedule in the submission. I'm hoping we'll be able to split participants up into groups of 3-4 who can discuss among themselves and also do teaching demos, somehow. But we'll get to know later which room we get

Reply:
At least in SC review, reviewers were looking very detailed in the technical implementation plan, so maybe we could reflect in the text that we are aware that this might be tricky but will adapt according to room assignment and technical possibilities?